### ISSUE TITLE
Bard Fields are not working

### ISSUE THREAD

**Issue by SG-fabian** [appswithlove/statamic-one-click-content-translation#5]
The plugin now works as expected - except for the Bard Fields, which are broken. Its an empty, non-editable block with icons on the right (see seconds screenshot).

<img width="584" alt="Bildschirmfoto 2023-02-20 um 10 44 48" src="https://user-images.githubusercontent.com/121227499/220070201-b39bf13f-72bf-4eb7-b35e-c6b7168fdf68.png">

<img width="913" alt="Bildschirmfoto 2023-02-20 um 10 54 30" src="https://user-images.githubusercontent.com/121227499/220072559-0e788aab-8070-4ab9-9dbf-ed4d19a52c01.png">

Statamic 3.4.4, PHP 8.1

I uninstalled the plugin and reinstalled it in the control panel just to make sure it did not miss anything.

**Comment by OleksiiBrylin** [comment]
Hi @SG-fabian. 
Have you tried to remove vendor assets from 'public/vendor/statamic-one-click-content-translation/*' and run 
php artisan vendor:publish --tag="statamic-one-click-content-translation" ? 

What is your addon version? The version should be v0.9.2.

**Comment by SG-fabian** [comment]
Hello,

i've removed the assets and published them again, but nothing changed. And it's version 0.9.2 :/

It's a fresh installation of the Statamic Peak with multi-site active, maybe that's a factor.

```
{
    "name": "statamic/statamic",
    "description": "Statamic",
    "keywords": [
        "statamic",
        "cms",
        "flat file",
        "laravel"
    ],
    "type": "project",
    "require": {
        "php": "^8.0",
        "appswithlove/statamic-one-click-content-translation": "^0.9.2",
        "fruitcake/laravel-cors": "^2.0.5",
        "guzzlehttp/guzzle": "^7.2",
        "laravel/framework": "^9.0",
        "laravel/sanctum": "^2.14",
        "laravel/tinker": "^2.7",
        "statamic/cms": "3.4.*",
        "studio1902/statamic-peak-browser-appearance": "^1.0",
        "studio1902/statamic-peak-commands": "^1.0",
        "studio1902/statamic-peak-seo": "^1.0",
        "studio1902/statamic-peak-tools": "^1.0"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.6",
        "fakerphp/faker": "^1.9.1",
        "laravel/sail": "^1.13",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^6.1",
        "phpunit/phpunit": "^9.5.10",
        "spatie/laravel-ignition": "^1.0"
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "composer/package-versions-deprecated": true,
            "pixelfear/composer-dist-plugin": true
        }
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "scripts": {
        "pre-update-cmd": [
            "Statamic\\Console\\Composer\\Scripts::preUpdateCmd"
        ],
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi",
            "@php artisan statamic:install --ansi"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ]
    }
}
```

**Comment by OleksiiBrylin** [comment]
I can reproduce the bug. It started appear from Statamic 3.4. 
As [Bard addons / extensions will no longer work](https://statamic.dev/upgrade-guide/3-3-to-3-4#bard-addons-extensions-will-no-longer-work).

**Comment by OleksiiBrylin** [comment]
@SG-fabian fixed in v0.9.4
